.travis.yml

```
@@ -5,25 +5,28 @@ language: python
 python:
   - 3.5
   - 3.6
+  - 3.7
+  - 3.8-dev
+
+dist: xenial
+sudo: required
 
-.mixins:
-- &_xenial
-  dist: xenial
-  sudo: required
 jobs:
   fast_finish: true
   allow_failures:
   - python: 3.8-dev
   include:
-  - python: 3.7
-    <<: *_xenial
-  - python: 3.8-dev
-    <<: *_xenial
   - stage: Deploy
     name: Publishing a source dist to PyPI
     python: 3.6
```

maybe update this too?
```suggestion
    python: 3.7
```